ROADMAP.md

```diff
@@ -2,105 +2,129 @@
+- Source repositories
+  - https://github.com/openebs/node-disk-manager
+- Backlogs
+  - [Done] Enhance the discovery probes to identify virtual storage (without WWN) moving across nodes
+  - [Done] Add gRPC API to list and re-scan block device
+  - [Done] Enhance the discovery probes to detect if the device already has device mapper, zfs and so forth
+  - [Done] Scan for device media errors and report them as prometheus metrics via ndm-exporter
+  - [Done] Label the block devices so that they can be reseved for use by different StorageClasses
+  - [In-progress] Auto-detecting capacity and mountpoint changes and updating the block device CR 
+  - [In-progress] Additional integration and e2e tests
+  - Support for using a custom node label to claim devices (instead of default kubernetes.io/hostname)
+  - Support Bulk BDC requests to claim multiple block devices that satisfy affinity or anti-affinity rules of applications. Example: two block devices from same node or two block devices from different nodes. 
+  - Support for device configuration tasks like partitioning, mounting or unmounting devices by adding new services via NDM gRPC API layer.
 
-### OpenEBS CLI
+
+### Others
 - Source repositories
+  - https://github.com/openebs/charts
   - https://github.com/openebs/openebsctl
+  - https://github.com/openebs/monitoring
+  - https://github.com/openebs/website
+  - https://github.com/openebs/openebs-docs
+  - https://github.com/openebs/maya
+  - https://github.com/openebs/m-exporter
+  - https://github.com/openebs/openebs-k8s-provisioner
 - Backlogs
-  - Add the existing functionality available in [mayactl](https://github.com/openebs/maya/tree/master/cmd/mayactl) for volume management to openebsctl
-  - Add Local PV volumes related status checks to openebsctl 
+  - [Done] Move towards GitHub actions based builds from Travis for all the repositories.
+  - [Done] Enable multi-arch builds.
+  - [Done] Add OpenAPI validations for the OpenEBS CRDs 
+  - [Done] Building additional Grafana Dashboards for OpenEBS Components, Block Devices, Pools and Volumes, that can be used to monitor SLOs
+  - [Done] Dashboard/UI for monitoring and managing cStor pools and volumes
+  - [Done] Split the provisioners and/or operators from the mono-repos [openebs/maya](https://github.com/openebs/maya) and [openebs/external-storage](https://github.com/openebs/external-storage) into individual repos
+  - [Done] Simplify the setup of NFS based Read-Write-Many volumes using OpenEBS RWO block volumes
+  - [Done] Add the existing functionality available in [mayactl](https://github.com/openebs/maya/tree/master/cmd/mayactl) for volume management to openebsctl
+  - [In-progress] Provide component level helm charts, that can then be used as dependent charts by the openebs chat
+  - [In-progress] Refactor the website and user documentation to be built as a single website using Hugo, similar to other CNCF projects 
+  - [In-progress] Add supoort for Kyerno, as a replacement for PSP
```

Comment:
Add supoort for Kyerno  -> Add support for Kyverno

@kmova Please find the above.